src/main/java/ch/admin/bag/covidcertificate/domain/VaccineImportControlRepository.java

```diff
@@ -0,0 +1,18 @@
+package ch.admin.bag.covidcertificate.domain;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+@Repository
+public interface VaccineImportControlRepository extends JpaRepository<VaccineImportControl, String> {
+
+    @Query(value = "select vic from VaccineImportControl vic " +
+            "where vic.importDate <= :importDate " +
+            "and vic.done = false")
+    Optional<VaccineImportControl> findByImportDate(@Param("importDate") LocalDate importDate);
```

Comment:
jpa does not need a query here - findByImportDateLessThanEqualAndDoneFalse gets translated natively to the right query (see https://www.bezkoder.com/jpa-repository-query/#JPA_Repository_Query_with_Comparison and https://www.bezkoder.com/jpa-repository-query/#JPA_Boolean_Query)